tests/functions/test_openai.py

```diff
@@ -4,10 +4,23 @@
 
 import pixeltable as pxt
 import pixeltable.exceptions as excs
+import pixeltable.functions as pxtf
 
 from ..utils import SAMPLE_IMAGE_URL, skip_test_if_not_installed, stock_price, validate_update_status
 
 
+@pxt.udf
+def stock_price_tool(ticker: str) -> str:
```

Comment:
I suggest using the existing `stock_price` in `tests/utils.py` which is shared among all the tool-calling tests.

There should be a `weather` defined there too; it's currently defined inline in various places to test that pickled UDFs work, but pickled UDFs are going away. e.g. sarch for `def weather` in `test_openai.py`.

Reply:
Done.